packages/TagPicker/src/TagPicker.tsx

```diff
@@ -126,6 +138,35 @@ ref: React.Ref<HTMLInputElement>) => {
         defaultKeyboardFocusIndex
     );
     const mergedInputRef = mergeRefs(inputRef, ref);
+    const listResults = React.useMemo(
+        () => results?.map(result => {
+            let listResultItem: OptionType = {
+                value: result.id,
+                label: result.text,
+                description: result.subtext,
+                type: "list",
+                color: result.color,
+                icon: result.icon,
+                src: result.src
+            };
+
+            if (result.type === "member") {
+                listResultItem = {
```

Comment:
listResultItem.type = "member";
listResultItem.manager = result.isManager;

simpler than duplicate everything?

Reply:
ya you have a point there :P